### PR TITLE
fix: filtering with account id

### PR DESCRIPTION
**Pull request by ceoaliongroo**
#344 
screen shot 2015-02-11 at 11 48 56 am

### CODE REVIEW

**Comment by ceoaliongroo** [comment]
@shahardo, @amitaibu how it's possible create a filter for a meter resource?
 
https://github.com/Gizra/negawatt-server/blob/master/negawatt/modules/custom/negawatt_restful/plugins/restful/node/meters/1.0/meter__1_0.inc

**Comment by shahardo** [comment]
http://localhost/negawatt-server/www/api/iec_meters?filter[account]=2

דר' שחר דולב
[image: phone.gif] 052-4750879
[image: greenchange.gif] <http://www.greenchange.co.il/profile/shr>[image:
Facebook] <http://www.facebook.com/profile.php?id=790744437>[image:
LinkedIn] <http://www.linkedin.com/in/shahard>[image: Twitter]
<http://twitter.com/shahard>[image: Picasa]
<http://picasaweb.google.com/shahar.dolev>[image: Delicious]
<http://delicious.com/shahard>[image: Plaxo]
<http://www.plaxo.com/profile/show/12885102543?src=name&pk=46c8d157f387b68e54d16c0fd41559cbf0cf9170>
[image: Skype/][image: Google Talk/] shahar.dolev

2015-02-22 11:38 GMT+02:00 Carlos Mantilla <notifications@github.com>:

**Comment by ceoaliongroo** [comment]
Already works in that way, The idea is moving to resource meter, right?

**Comment by shahardo** [comment]
Problem is fixed on branch 375.

File: negawatt/modules/custom/negawatt_restful/plugins/restful/node/meters/1.0/NegawattMeterResource.class.php [hunk 1]
```diff
@@ -7,4 +7,53 @@
 class NegawattMeterResource extends \RestfulEntityBaseMultipleBundles {
   // Will automatically redirect Satec meters to NegawattSatecMeterResource.class.php
   // and IEC meters to NegawattIecMeterResource.class.php
+
```
Comment (ceoaliongroo) [inline negawatt/modules/custom/negawatt_restful/plugins/restful/node/meters/1.0/NegawattMeterResource.class.php]: This is need it, to do the filtering by field example:
``http://localhost/negawatt-server/www/api/meters?filter[account]=2``

**Comment by amitaibu** [comment]
Merged.

**Pull request closed without merging**